feature-libs/organization/administration/core/http-interceptors/conflict/conflict.handler.ts

```diff
@@ -25,26 +25,33 @@ export class OrganizationConflictHandler extends HttpErrorHandler {
 
   handleError(request: HttpRequest<any>, response: HttpErrorResponse) {
     return this.getErrors(response).forEach(({ message }: ErrorModel) => {
-      // Handle budget conflict
-      this.handleConflict(message, this.budgetMask, 'budget');
-      // Handle user email conflict
-      this.handleConflict(message, this.userMask, 'user', request?.body?.email);
-      // Handle user group conflict
-      this.handleConflict(
-        message,
-        this.userGroupMask,
-        'userGroup',
-        request?.body?.uid
-      );
-      // Handle unit conflict
-      this.handleConflict(message, this.unitMask, 'unit');
+      if (message) {
+        // Handle budget conflict
+        this.handleConflict(message, this.budgetMask, 'budget');
+        // Handle user email conflict
+        this.handleConflict(
+          message,
+          this.userMask,
+          'user',
+          request?.body?.email
+        );
+        // Handle user group conflict
+        this.handleConflict(
+          message,
+          this.userGroupMask,
+          'userGroup',
+          request?.body?.uid
+        );
+        // Handle unit conflict
+        this.handleConflict(message, this.unitMask, 'unit');
+      }
     });
   }
 
   protected matchMask(response: HttpErrorResponse): boolean {
     return this.getErrors(response).some((error) =>
       [this.budgetMask, this.userMask, this.userGroupMask, this.unitMask].some(
-        (mask) => mask.test(error.message)
+        (mask) => mask.test(error.message ?? '')
```

Comment:
method protected handleConflict is missing a return type : void. Since we're making sure all types are tidy and strict....